src/managing-workflow/production-deployments.md

```diff
@@ -1,3 +1,57 @@
 # Production Deployments
 
-TODO (bacongobbler): rewrite for v2
+Many Workflow users are running Workflow quite successfully in production. When readying a Workflow
+deployment for production workloads, there are some additional (but optional) recommendations.
+
+
+## Running Workflow without Minio
+
+Workflow makes use of [Minio][] to provide persistent storage for the [Registry][], [Database][],
+and [Logger][] components. The additional operational complexity of Minio is tolerated because of
+the need for persistent storage for platform high availability.
+
+Alternatively, persistent storage can be achieved by running an external S3-compatible blob store,
+PostgreSQL database, and log service. For users on AWS, the convenience of Amazon S3 and Amazon RDS
+make the prospect of running a Minio-less Workflow cluster quite reasonable.
+
+Running a Workflow cluster without Minio provides several advantages:
+
+ - Removal of state from the control plane (etcd is still used for configuration)
+ - Reduced resource usage
+ - Reduced complexity and operational burden of managing Workflow
+
+See [Configuring Object Storage][] for details on removing this operational complexity.
+
+
+## Review Security Considerations
+
+There are some additional security-related considerations when running Workflow in production, and
+users can consider enabling a firewall on the CoreOS hosts as well as the router component.
+
+See [Security Considerations][] for details.
+
+
+## Change Registration Mode
+
+Changing the registration process is highly recommended in production. By default, registrations
+for a new cluster are open to anyone with the proper URL. Once the admin user has registered with a
+new cluster, it is recommended that you either turn off registrations entirely or enable the admin
```

Comment:
I would say instead:
> it is recommended either to disable registrations or to enable the admin-only registration feature.

Specifically, I prefer avoiding "you" when possible, and "admin-only" should contain a hyphen.